UpsertedProfileOrchestrator/handler.ts

```diff
@@ -202,48 +203,91 @@ export const getUpsertedProfileOrchestratorHandler = (params: {
         } as UpdateServiceSubscriptionFeedActivityInput
       );
     } else {
-      // When the profile gets updates, we extract the services that have been
-      // blocked and unblocked during this profile update.
-      // Blocked services get mapped to unsubscribe events, while unblocked ones
-      // get mapped to subscribe events.
-      const {
-        e1: unsubscribedServices,
-        e2: subscribedServices
-      } = diffBlockedServices(
-        oldProfile.blockedInboxOrChannels,
-        newProfile.blockedInboxOrChannels
-      );
-
-      for (const s of subscribedServices) {
-        context.log.verbose(
-          `${logPrefix}|Calling UpdateSubscriptionsFeedActivity|OPERATION=SUBSCRIBED|SERVICE_ID=${s}`
-        );
-        yield context.df.callActivityWithRetry(
-          "UpdateSubscriptionsFeedActivity",
-          retryOptions,
-          {
-            fiscalCode: newProfile.fiscalCode,
-            operation: "SUBSCRIBED",
-            serviceId: s as ServiceId,
-            subscriptionKind: "SERVICE",
-            updatedAt: updatedAt.getTime(),
-            version: newProfile.version
-          } as UpdateServiceSubscriptionFeedActivityInput
+      const { newServicePreferencesMode, oldServicePreferenceMode } = {
+        newServicePreferencesMode: newProfile.servicePreferencesSettings.mode.toString(),
+        oldServicePreferenceMode: oldProfile.servicePreferencesSettings.mode.toString()
+      };
```

Comment:
```suggestion
        newServicePreferencesMode = newProfile.servicePreferencesSettings.mode.toString();
        oldServicePreferenceMode = oldProfile.servicePreferencesSettings.mode.toString();
```

Reply:
Yes it's for avoid writing two instructions ;)

Reply:
I know :)